ui/components/component-library/avatar-network/README.mdx

```diff
@@ -34,34 +34,82 @@ Defaults to `md`
   <Story id="ui-components-component-library-avatar-network-avatar-network-stories-js--size" />
 </Canvas>
 
-###  Network Name
+```jsx
+import { AvatarNetwork } from '../../component-library';
+  <AvatarNetwork size={SIZES.XS} />
+  <AvatarNetwork size={SIZES.SM} />
+  <AvatarNetwork size={SIZES.MD} />
+  <AvatarNetwork size={SIZES.LG} />
+  <AvatarNetwork size={SIZES.XL} />
+```
 
-Use the `networkName` prop to set the initial alphabet of the `AvatarNetwork`. This will be used as the fallback display if no image url is passed to the `networkImageUrl` prop.
+###  Name
+
+Use the `name` prop to set the initial alphabet of the `AvatarNetwork`. This will be used as the fallback display if no image url is passed to the `src` prop.
```

Comment:
grammer
```suggestion
Use the `name` prop to set the initial letter of the `AvatarNetwork`. This will be used as the fallback display if no image url is passed to the `src` prop.
```